deb/openmediavault/var/www/openmediavault/js/omv/Rpc.js

```diff
@@ -167,7 +168,11 @@ Ext.define("OMV.Rpc", {
 				me.setDisabled(true);
 				// Display a dialog forcing user to click 'OK' to reload
 				// the page.
-				OMV.MessageBox.guru({
+				if (this.guru)
+				{
+					this.guru.destroy();
+				}
+				this.guru = OMV.MessageBox.guru({
```

Comment:
I'm wondering why you need to take care to destroy the object? The window is created once, then the code path is never called, see me.setDisabled(true). So there is only one instance until the user clicks the mouse button. After that the whole page is reloaded.